src/pybind/mgr/dashboard/frontend/src/app/shared/helpers/helpers.module.ts

```diff
@@ -0,0 +1,8 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule]
+})
+export class HelpersModule {}
```

Comment:
Is this module being used?

Reply:
I couldn't find a place to put the prometheus-helpers anywhere so i created this helper module to put the prometheus-list-helper class inside. Although we are not directly accessing the module, the prometheus-list-helper gets extended to the DashboardComponent and couple of others as well.